qiskit/circuit/library/standard_gates/r.py

```diff
@@ -24,34 +24,30 @@
 class RGate(Gate):
     """Rotation θ around the cos(φ)x + sin(φ)y axis."""
 
-    def __init__(self, theta, phi):
+    def __init__(self, theta, phi, phase=0):
         """Create new r single-qubit gate."""
-        super().__init__('r', 1, [theta, phi])
+        super().__init__('r', 1, [theta, phi], phase=phase)
 
     def _define(self):
         """
         gate r(θ, φ) a {u3(θ, φ - π/2, -φ + π/2) a;}
         """
         from .u3 import U3Gate
-        definition = []
         q = QuantumRegister(1, 'q')
         theta = self.params[0]
         phi = self.params[1]
-        rule = [
-            (U3Gate(theta, phi - pi / 2, -phi + pi / 2), [q[0]], [])
+        self.definition = [
+            (U3Gate(theta, phi - pi / 2, -phi + pi / 2, phase=self.phase), [q[0]], [])
```

Comment:
i dont think it is a good idea to make phase change. For R it is defined with phase 0